common/JailUtil.cpp

```diff
@@ -581,13 +581,13 @@ bool updateDynamicFilesImpl(const std::string& sysTemplate)
 
         if (checkWritableSysTemplate && !FileUtil::isWritable(sysTemplate))
         {
-            disableBindMounting(); // We can't mount from incomplete systemplate that can't be updated.
```

Comment:
This assumes we mount /etc on-top of the mounted sysTemplate? I trust that is working as expected.

Reply:
I certainly believe so, and it does for me when I try it. [This is the existing bit that does it](https://github.com/CollaboraOnline/online/commit/852e8545af3f1807ff6619fe52a922f35bbd9da4#diff-a835d79df212a8e0dbe2430c2901a1e8dc94086e785f6fd52f269b80009a1d11R3063)